{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -17,15 +17,15 @@ commands = flake8 {{ cookiecutter.project_slug }}
 [testenv]
 setenv =
     PYTHONPATH = {toxinidir}
-{% if cookiecutter.use_pytest == 'y' -%}
 deps =
     -r{toxinidir}/requirements_dev.txt
 ; If you want to make tox run the tests with the same versions, create a
 ; requirements.txt with the pinned versions and uncomment the following line:
 ;     -r{toxinidir}/requirements.txt
+{% if cookiecutter.use_pytest == 'y' -%}
 commands =
     pip install -U pip
     pytest --basetemp={envtmpdir}
```

Comment:
Above, the command was `pytest ./tests`

Reply:
I moved this down because requirements_dev.txt needs to be installed irrespective of which test suite is in use.